WooCommerce/Classes/ViewRelated/Blaze/CampaignCreation/BlazeCampaignCreationFormViewModel.swift

```diff
@@ -433,10 +438,10 @@ private extension BlazeCampaignCreationFormViewModel {
     // Updates the `isUsingAISuggestions` property based on whether the current
     // `tagline` and `description` match any of the provided AI suggestions.
     // The property will be set to `true` if there is at least one suggestion
-    // that matches both the `tagline` and `description`, and the suggestions list is not empty.
+    // that matches both the `tagline` and `description`.
     func updateIsUsingAISuggestions() {
         isUsingAISuggestions = suggestions.contains { element in
-            element.siteName == tagline && element.textSnippet == description && !suggestions.isEmpty
```

Comment:
the `!suggestions.isEmpty` was redundance, the `contains` will never be `true` if the list is empty, so I removed it for simplicity.